utils/appliance/endpoints/ui.py

```diff
@@ -86,7 +86,7 @@ def pre_navigate(self, _tries=0):
             logger.debug(store.current_appliance.ssh_client.run_command(
                 'top -c -b -n1 -o "%MEM" | head -30').output)  # noqa
             logger.debug('Managed Providers:')
-            logger.debug(store.current_appliance.managed_providers)
+            logger.debug([prov.key for prov in store.current_appliance.managed_providers])
```

Comment:
Or we just need to accept that passing around full crud objects simply has this downside.